WikiWatershed/model-my-watershed
# Collections API

## Overview

Formally includes the Collections API work by merging the feature branch. Includes #2100, #2101, #2102, #2103, #2104, https://github.com/WikiWatershed/mmw-geoprocessing/issues/48, https://github.com/WikiWatershed/mmw-geoprocessing/issues/49, https://github.com/WikiWatershed/mmw-geoprocessing/issues/50, https://github.com/WikiWatershed/mmw-geoprocessing/issues/51, https://github.com/WikiWatershed/mmw-geoprocessing/issues/52, https://github.com/WikiWatershed/mmw-geoprocessing/issues/53.

All the code in this PR has already been reviewed. This only requires a quick run through, before merging.

Connects #2105 

### Notes

Some areas of interest will not work for MapShed, because of an alignment mismatch issue described in #2153. To test the values here outside of that, try and choose a horizontal area of interest, and not a diagonal one. These HUC-12's should work:

![image](https://user-images.githubusercontent.com/1430060/29890997-90133a82-8d97-11e7-9ee1-6e06a83d6ac0.png)

## Testing Instructions

 * Check out this branch. Destroy your worker, and reprovision it.
 * Run through the app. Ensure you can still analyze and model.
 * Ensure TR-55 works. Compare results against staging and ensure they are identical.
 * Ensure MapShed works. Compare results against staging and ensure they are identical.
